.github/collector/docker-compose.yml

```diff
@@ -23,6 +23,8 @@ services:
       - AWS_SESSION_TOKEN
       - OTEL_RESOURCE_ATTRIBUTES=service.name=aws-otel-integ-test
       - OTEL_EXPORTER_OTLP_ENDPOINT=http://otel:4317
```

Comment:
can we change to :4318, to follow the default recommendation using http/protobuf?